app/misc/git.ts

```diff
@@ -256,6 +256,28 @@ function clearCommitMessage() {
   document.getElementById('commit-message-input').value = "";
 }
 
+// checking if the length of commits is different
+function checkCommitChange() {
```

Comment:
This check isn't complete as the graph can change while the # of commits may stay the same. I don't know how it would be possible to truly detect changes in the graph without fully analyzing the structure of the graph. Perhaps it might just be easier to (perhaps in addition to these changes) provide a button to allow the user to refresh manually?

Reply:
@sterlinglaird This implementation is indeed very sloppy, I can't think of any solution that could do the full check without using too much resource. I will include a refresh button.